A test assertion fails on EPEL 7 due to str != unicode

This test seems to expect a string where we are passing a unicode. This is likely due to my recent change to try to use unicode in many places in the tests:

```
======================================================================
FAIL: Ensure we cannot submit an older build to stable when a newer one
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/mock.py", line 1201, in patched
    return func(*args, **keywargs)
  File "/builddir/build/BUILD/bodhi-6a0a0ff8eafda220ae4d1fc4a560ffe1de2889d8/bodhi/tests/server/services/test_updates.py", line 3263, in test_submit_older_build_to_stable
    status=400)
  File "/usr/lib/python2.7/site-packages/webtest/app.py", line 821, in post_json
    content_type=content_type)
  File "/usr/lib/python2.7/site-packages/webtest/app.py", line 783, in _gen_request
    expect_errors=expect_errors)
  File "/usr/lib/python2.7/site-packages/webtest/app.py", line 1043, in do_request
    res = req.get_response(app, catch_exc_info=True)
  File "/usr/lib/python2.7/site-packages/WebOb-1.4.1-py2.7.egg/webob/request.py", line 1313, in send
    application, catch_exc_info=True)
  File "/usr/lib/python2.7/site-packages/WebOb-1.4.1-py2.7.egg/webob/request.py", line 1281, in call_application
    app_iter = application(self.environ, start_response)
  File "/usr/lib/python2.7/site-packages/webtest/lint.py", line 146, in lint_app
    check_environ(environ)
  File "/usr/lib/python2.7/site-packages/webtest/lint.py", line 330, in check_environ
    % (key, type(environ[key]), environ[key]))
AssertionError: Environmental variable PATH_INFO is not a string: <type 'unicode'> (value: u'/updates/bodhi-1.0-1.fc17/request')
```

--- a/bodhi/tests/server/services/test_updates.py
+++ b/bodhi/tests/server/services/test_updates.py
@@ -3258,7 +3258,7 @@ class TestUpdatesService(bodhi.tests.server.functional.base.BaseWSGICase):
 
         # Try and submit an older build to stable
         resp = self.app.post_json(
-            '/updates/%s/request' % oldbuild,
+            '/updates/%s/request' % str(oldbuild),
             {'request': 'stable', 'csrf_token': self.get_csrf_token()},
             status=400)
         eq_(resp.json['status'], 'error')

